priv/rel/files/boot

```diff
@@ -187,10 +200,10 @@ generate_sys_config() {
                 __conform="$ROOTDIR/bin/conform"
             fi
 
-            # remove old sys.config since conform seems to merge this one too
-            rm -v "$GENERATED_CONFIG_DIR/sys.config"
+            # remove old sys.config first from the mutable dir since conform also considers this file for merging
+            rm -fv $SYS_CONFIG
```

Comment:
no, there are cases where you dont want to remove a vm.args, e.g. 'restart'. That's the reason I left this.